handlers.go

```diff
@@ -403,7 +404,7 @@ func (r *oauthProxy) logoutHandler(w http.ResponseWriter, req *http.Request) {
 	}
 
 	// set the default revocation url
-	revokeDefault := fmt.Sprintf("%s/protocol/openid-connect/logout", strings.TrimSuffix(r.config.DiscoveryURL, "/.well-known/openid-configuration"))
+	revokeDefault := fmt.Sprintf("%s/protocol/openid-connect/revoke", strings.TrimSuffix(r.config.DiscoveryURL, "/.well-known/openid-configuration"))
 	revocationURL := defaultTo(r.config.RevocationEndpoint, revokeDefault)
```

Comment:
Can we move these down to just above line 429 where we use `revocationURL`?

Reply:
ok